internal/releasesources/clients.go

```diff
@@ -0,0 +1,146 @@
+package releasesources
```

Comment:
I assume this sort of ocm client?

Reply:
There is no OCM client provided by ocm?

Reply:
I did not find any. There is an OCM client for golang but it doesn't provide a way to reach `OpenShift Cincinnati`